tests/release/pipelines/fbc_release.go

```diff
@@ -223,7 +224,25 @@ func assertReleasePipelineRunSucceeded(devFw, managedFw framework.Framework, dev
 		return nil
 	}, 5 * time.Minute, releasecommon.DefaultInterval).Should(Succeed(), "timed out when waiting for Snapshot and Release being created")
 
-	mFw := releasecommon.NewFramework(managedWorkspace)
+	mFw = releasecommon.NewFramework(managedWorkspace)
+	// Create a ticker that ticks every 3 minutes
+	ticker := time.NewTicker(3 * time.Minute)
+	// Schedule the stop of the ticker after 15 minutes
+	time.AfterFunc(15*time.Minute, func() {
+		ticker.Stop()
+		fmt.Println("Stopped executing every 3 minutes.")
+	})
+	// Run a goroutine to handle the ticker ticks
+	go func() {
+		for {
```

Comment:
the linter did not like this. I found this, might help to fix: https://github.com/dominikh/go-tools/issues/503#issuecomment-1632211731

Reply:
Thanks!  Updated.